plugins/postgresql/postgresql.go

```diff
@@ -4,17 +4,21 @@ import (
 	"bytes"
 	"database/sql"
 	"fmt"
+	"regexp"
 	"strings"
 
 	"github.com/influxdb/telegraf/plugins"
 
-	_ "github.com/lib/pq"
+	"github.com/lib/pq"
 )
 
 type Postgresql struct {
 	Address        string
 	Databases      []string
 	OrderedColumns []string
+
+	VerbatimAddress  bool
```

Comment:
Let's just remove the `VerbatimAddress` option and make it the default.

If people were relying on tag values that contain their password, they really should take the pain and migrate to just the server URL.